cmds/core/sshd/sshd.go

```diff
@@ -63,13 +63,22 @@ func runCommand(c ssh.Channel, p *pty.Pty, cmd string, args ...string) error {
 		ps, _ = p.C.Process.Wait()
 	} else {
 		e := exec.Command(cmd, args...)
-		e.Stdin, e.Stdout, e.Stderr = c, c, c
 		log.Printf("Executing non-PTY command %s %v", cmd, args)
-		if err := e.Start(); err != nil {
+		// execute command and wait for response
+		b, err := e.CombinedOutput()
```

Comment:
I think this is the wrong approach. For example, suppose my command is this:
dd if=/dev/hugedisk # disk is 50x the size of memory
If you use combined output, you will not be able to properly run the command, as it will have to be buffered on the sshd machine.

Reply:
Yeah, this is a good point. I will disable output check in test for now to make CI stable. And will try to come up with new solution.

Reply:
Before removing part of the test, I made another attempt, not sure why but this seems works on my cpu-shared system and I don't see EOF error. I changes Start to Run and remove c.Stdin because otherwise everything hangs. Previously execution was asynchronous and wait condition was on e.Process.Wait() and I thought that sshd may return early then send response back through the channel and time.Sleep was fixing issue as well.